README.md

```diff
@@ -1,46 +1,163 @@
 # Catalyst
 
+The entire bundle of application needed for booting up livepeer's
+decentralised media server in place is termed as [catalyst].
+
 ## Getting Started
 
-go1.17 is recommended because building with < 1.17 (specifically 1.16) can result in issue on arm64 architectures.
+Make sure you have the following installed on the machine before
+proceeding:
 
-From there:
+  - go1.17± or newer
```

Comment:
@AlexKordic @emranemran go1.17 is only needed for the scripts in catalyst project. For any dependent ones, either the prebuilt binaries can be used or each projects' own dependency tree would be needed (as mentioned later in readme).

Should I move that clause further up, to avoid confusion? as cmake, gcc, git etc. requirements would also pop-up for eg. when trying to build MistServer etc.